libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/item/IpvSessionItem.java

```diff
@@ -37,6 +38,10 @@ public class IpvSessionItem implements PersistenceItem {
     private ReverificationStatus reverificationStatus;
     private List<String> stateStack = new ArrayList<>();
 
+    // These are used as part of an unsuccessful reverification response
+    private ReverificationFailureCode failureCode;
+    private String failureDescription;
```

Comment:
Is the description ever variable - could we just store the code and put the description in the enum as well?